recipes/beniget/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "beniget" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 509a8f67e397f209ce6f3f97dc78a15c4d0e8798b194b4122bec82e44bb26bf2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - gast >=0.2.2
+    - python
+
+test:
+  imports:
+    - beniget
+
+about:
+  home: https://github.com/serge-sans-paille/beniget/
+  license: BSD3
```

From https://opensource.org/licenses/alphabetical:
```suggestion
  license: BSD-3-Clause
```